readthedocs/api/v2/views/footer_views.py

```diff
@@ -232,7 +232,14 @@ def get(self, request, format=None):
             'version_supported': version.supported,
         }
 
-        return Response(resp_data)
+        response = Response(resp_data)
+        cache_tags = [
+            project.slug,
+            f'{project.slug}-{version.slug}',
+            f'{project.slug}-rtd-footer',
```

Comment:
we could change this to an impossible version slug like `rtd--footer`, but I'm not too worried about that

Reply:
Why are we adding this custom cache-tag? What does it allow us to do that the project-level and version-level ones don't?

Reply:
There are cases where we only want to purge the footer (across all versions), so instead of purging the whole project, we just purge the footer. The footer changes across all versions more often (on every version change).

Reply:
Ah, I see in the other PR 👍